knife/lib/chef/knife/bootstrap.rb

```diff
@@ -306,6 +306,11 @@ class Bootstrap < Knife
         description: "Location of the #{ChefUtils::Dist::Infra::PRODUCT} MSI. The default templates will prefer to download from this location. The MSI will be downloaded from #{ChefUtils::Dist::Org::WEBSITE} if not provided (Windows).",
         default: ""
 
+      option :bootstrap_certstore_cert,
+        long: "--bootstrap-certstore-cert",
```

Comment:
This needs to be windows only since we don't support macos yet

Reply:
done